doc/nrf/ug_thingy91.rst

```diff
@@ -161,49 +161,64 @@ You can choose the method based on the availability or absence of an external de
+   If you installed the |NCS| using the :ref:`gs_app_tcm`, you can click the :guilabel:`Open VS Code` button next to version you installed.
 
-..
+#. Click :guilabel:`Create new application from sample` from the :guilabel:`Welcome` panel.
 
-   .. figure:: images/ses_thingy_configuration.png
-      :alt: Opening the Asset tracker v2 application
+   The :guilabel:`New Application` screen opens.
 
-      Opening the Asset tracker v2 application for the thingy91_nrf9160_ns build target
+#. Provide the following information to create the application:
 
-   .. note::
+   * For :guilabel:`Application type`, select :guilabel:`Freestanding`.
+     This links the application to the version of |NCS| you have installed.
+   * Select the |NCS| version from the :guilabel:`nRF Connect SDK` drop-down menu and the corresponding toolchain version form the :guilabel:`nRF Connect Toolchain` drop-down menu.
+     The version you selected when setting up |VSC| is selected by default.
+   * Choose the location where the application is saved.
+     To do this, either type the path in the :guilabel:`Application location` field, or browse to the desired location by clicking the :guilabel:`...` button.
+   * Select the application template that you want to use.
 
-      The *Board Directory* folder can be found in the following location: ``ncs/nrf/boards/arm``.
+     You can either select one from the :guilabel:`Application template` drop-down menu, or browse the available templates with a preview of their documentation in the :guilabel:`Browse Samples` screen by clicking the :guilabel:`Select...` button.
+     Both the drop-down and :guilabel:`Browse Samples` screen contain the same applications.
+   * Fill in the :guilabel:`Application name`.
 
-4. Click :guilabel:`OK` to import the project into SES.
-   You can now work with the project in the IDE.
+     When you create the application, this creates a new folder in the :guilabel:`Application folder` directory with the same name.
 
-   .. include:: gs_programming.rst
-      :start-after: build_SES_projimport_note_start
-      :end-before: build_SES_projimport_note_end
+#. Click the :guilabel:`Create Application` button.
 
-#. To build the sample or application:
+   The selected application appears in the :guilabel:`Applications` panel.
 
-   a. Select your project in the Project Explorer.
-   #. From the menu, select :guilabel:`Build` > :guilabel:`Build Solution`.
-      This builds the project.
+#. Click the :guilabel:`No build configurations. *Click to create one*` text under the application.
 
-   You can find the output of the build, which includes the merged HEX file containing both the application and the SPM, in the ``zephyr`` subfolder in the build directory.
+   The :guilabel:`Add Build Configuration` screen opens.
 
-.. prog_extdebugprobe_start
+#. Provide the following information to configure the build:
+
+   * Select the build target from the :guilabel:`Board` drop-down menu.
```

Comment:
Shouldn't these steps be in numeric order or the order of the alphabet as these are steps?